org/org.go

```diff
@@ -0,0 +1,27 @@
+package org
+
+import (
+	"encoding/json"
+
+	model "github.com/owasp-amass/open-asset-model"
+)
+
+// Organization represents an organization. Does not have to be tied with an RIR.
+type Organization struct {
+	// OrgName is the name of the organization.
+	OrgName string `json:"org_name,omitempty"`
+	// Industry is the industry of the organization.
+	Industry string `json:"industry,omitempty"`
+	// IANAid is the IANA ID of the organization.
+	IANAid string `json:"iana_id,omitempty"`
```

Comment:
Debating if this belongs here. Do we want this on the top level org-id or in the whois?

Reply:
We should probably take it out until we're sure about the correct location for the field

Reply:
After digging through some sites, like https://knowledge.whoisxmlapi.com/what-is-registrar-iana-id, it says that "Registrar IANA ID is a unique number allocated by ICANN to its accredited registrars.". This made it seem like this is unique to the registrar, so it may be redundant to include the same ID for each whois record associated with the same registrar.

I may be wrong, so feel free to correct me if so. But that was my thought process 😄

Reply:
we are going to move this to a `Registrar` record